lib/logstash/outputs/tcp.rb

```diff
@@ -38,6 +38,9 @@ class LogStash::Outputs::Tcp < LogStash::Outputs::Base
   # event will be written as a single line.
   config :message_format, :validate => :string, :deprecated => true
 
+  # respond_to? check needed for backwards compatibility with < 2.2 Logstashes
+  declare_workers_not_supported! if self.respond_to?(:workers_not_supported!)
```

Comment:
@jsvd I do see a bug here though, it should be `declare_workers_not_supported! if self.respond_to?(:declare_workers_not_supported!)`. I need to fix that in the asciidocs as well.